daemon/daemon_utils.go

```diff
@@ -60,7 +60,7 @@ func initialRuntime(baseDir string, runtimes map[string]types.Runtime) error {
 }
 
 // checkSnapshotterValid checks whether the given snapshotter is valid
-func checkSnapshotterValid(snapshotter string, ctrdClient ctrd.APIClient) error {
+func checkSnapshotterValid(snapshotter string, ctrdClient ctrd.APIClient, allowMultiSnapshotter bool) error {
```

Comment:
I think this function should be encapsulated into the package `ctrd` to be 
```
func (c *Client)  checkSnapshotterValid(snapshotter string,allowMultiSnapshotter bool) error
```

WDYT? @wangforthinker

Reply:
Fix it!